src/ChattingPage.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState, useEffect, useCallback } from 'react';
+import Top from './Top';
+import Chat from './Chat';
+import Form from './Form';
+import styled from 'styled-components';
+import ChatData from './ChatData.json';
+import {initialList} from './App.js';
+
+const ChattingPageContainer = styled.div`
+    margin-left: 70px;
+    display: flex;
+    flex-direction: column;
+    height: 550px;
+`;
+
+const ChattingPage = ({ match }) => {
+    const [isMe, setIsMe] = useState(true);
+    const [userName, setUserName] = useState('');
+    const [chatting, setChatting] = useState([]);
+    
+    useEffect(() => {
+        setChatting(ChatData[match.params.id]);
+        setUserName(initialList.find(e => e.id === match.params.id).name);
+    }, []);
+
+    const onClickTop = useCallback(() => {
```

Comment:
```suggestion
    const handleTopClick = useCallback(() => {
```

네이밍은 
prop 이름: on + Event
핸들러: on + Component +Event

이 표준이랍니다